test/unittest/tests/wrappers/mock_session.cpp

```diff
@@ -15,6 +15,13 @@
 MockSession::MockSession(WT_SESSION_IMPL *session, std::shared_ptr<MockConnection> mockConnection)
     : _sessionImpl(session), _mockConnection(std::move(mockConnection))
 {
+    // We could initialize this in the list but order is important and this seems easier.
+    WT_CLEAR(_handler_wrap.handler);
+    _handler_wrap.handler.handle_error = handleWiredTigerError;
+    _handler_wrap.handler.handle_message = handleWiredTigerMessage;
+    _handler_wrap.mock_session = this;
```

Comment:
We could replace this code with something like:
```
    _handler_wrap.handler = {handleWiredTigerError, handleWiredTigerMessage, nullptr, nullptr, nullptr};
    _handler_wrap.mock_session = this;
```

Reply:
[51ed2a5](https://github.com/wiredtiger/wiredtiger/pull/10968/commits/51ed2a51d3b53a7cc0d2796b05d41dbfd0c2ac96)